docs/modules/ROOT/nav.adoc

```diff
@@ -64,6 +64,7 @@ include::user:ROOT:partial$shared-nav.adoc[]
 ** xref:how-to/list-namespaces.adoc[List Namespaces]
 ** xref:how-to/using-templates.adoc[Using OpenShift Templates]
 ** xref:how-to/grant-access.adoc[Granting Access]
+** xref:how-to/fine-grained-access-examples.adoc[Fine grained access control examples]
```

Comment:
```suggestion
** xref:how-to/fine-grained-access-examples.adoc[]
```

This is a shortcut for when the title of the page is the same as in the navigation.